smac/runhistory/runhistory.py

```diff
@@ -314,7 +323,7 @@ def update_from_json(self, fn, cs):
         """
         new_runhistory = RunHistory(self.aggregate_func)
         new_runhistory.load_json(fn, cs)
-        self.update(runhistory=new_runhistory)
+        self.update(runhistory=new_runhistory, external_data=True)
```

Comment:
Okay, as we do not want external data to be written to disk, how about making the bool an integer (or an enum) and giving it the following values:

* 0: internal data, written to disk and used by the model and intensify
* 1: external data, not written to disk but used by the model and intensify
* 2: external data, not written to disk, only used by the model

This shouldn't take more space than the current implementation and keep the code relatively simple.